### PR TITLE
27180 css nesting

### PR DESCRIPTION
**Pull request by dletorey**
## Summary

I have been documenting CSS Nesting module and needed to add some pages to the menu via the `kumascript/macros/<% CSSRef.ejs` file

### Problem

N/A

### Solution

N/A
---

## Screenshots

N/A

### Before

N/A

### After

N/A

---

## How did you test this change?

This is related to the [Content PR](https://github.com/mdn/content/pull/28473), without this PR the pages will **not** be available for the menu items being added, so please check that this has been merged first.

### CODE REVIEW

**Comment by dletorey** [comment]
The [Content](https://github.com/mdn/content/pull/28473) & [BCD](https://github.com/mdn/browser-compat-data/pull/20523) PRs have now been merged so if this could also be merged that would be great

File: kumascript/macros/CSSRef.ejs [hunk 1]
```diff
@@ -125,6 +125,10 @@ const text = mdn.localStringMap({
         'Using_media_queries': 'Using media queries',
         'Using_media_queries_for_accessibility': 'Using media queries for accessibility',
         'Testing_media_queries_programmatically': 'Testing media queries programmatically',
+      'Nesting': 'Nesting',
```
Comment (OnkarRuikar) [inline kumascript/macros/CSSRef.ejs:128]: How about?
```suggestion
      'Nesting': 'Nesting style rules',
```

File: kumascript/macros/CSSRef.ejs [hunk 2]
```diff
@@ -1329,6 +1333,16 @@ async function buildPropertylist(pages, title) {
           </ol>
       </details>
   </li>
+  <li class="toggle">
+      <details>
+          <summary><%=text['Nesting']%></summary>
+          <ol>
+            <li><%-smartLink(`${cssURL}CSS_Nesting/Using_CSS_nesting`, null, text['Using_CSS_nesting'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_Nesting/Nesting_and_specificity`, null, text['Nesting_and_specificity'], cssURL)%></li>
```
Comment (OnkarRuikar) [inline kumascript/macros/CSSRef.ejs:1341]: We don't have to follow alphabetical order at this level. Do you wish to have the same [order mentioned on the module page](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting#guides)?

**Review by OnkarRuikar — APPROVED** [review]
This looks good to me. With or without the following suggestions.

File: kumascript/macros/CSSRef.ejs (hunk 2 above)
Comment (dletorey) [reply kumascript/macros/CSSRef.ejs]: I have switched this order.

**Review by OnkarRuikar — APPROVED** [review]

**Pull request merged**